src/QMCDrivers/WalkerControlBase.cpp

```diff
@@ -385,11 +387,8 @@ int WalkerControlBase::sortWalkers(MCWalkerConfiguration& W)
   //W.EnsembleProperty.Variance=(e2sum*wsum-esum*esum)/(wsum*wsum-w2sum);
   if (!WriteRN)
   {
-    if(good_w.empty())
-    {
-      app_error() << "All the walkers have died. Abort. " << std::endl;
-      APP_ABORT("WalkerControlBase::sortWalkers");
-    }
+#if 0
```

Comment:
If this code is not needed anymore, it should be deleted entirely.
If this code should be left in place, the formatting changes below should be undone.  Otherwise it shows up in the diff (and it's a bit confusing to see diffs in code that is not executed.  It took a little time to see that the only change was formatting.  It also didn't help that the diff algorithm performed poorly on this change and displayed a bunch of lines as deleted and added)

Reply:
@ye-luo please can you address this?